### PR TITLE
Update use-network-upload-to-import-pst-files.md

### PR DESCRIPTION
**Pull request by ShraddhaSSwadi**
The Azure Storage Explorer desktop application is generally available. We need to ensure that customers are aware of it. The Storage Explorer available on the Azure Portal, is currently in preview.

### CODE REVIEW

**Comment by ShraddhaSSwadi** [comment]
The Azure Storage Explorer desktop application is generally available. We need to ensure that customers are aware of it. The Storage Explorer available on the Azure Portal, is currently in preview.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/use-network-upload-to-import-pst-files.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ As an optional step, you can install and use the Microsoft Azure Storage Explore
 
 - Verify the filename (and the subfolder pathname if you included one) for each PST file uploaded to the Azure blob. This is helpful when you're creating the PST mapping file in the next step because you have to specify both the folder pathname and filename for each PST file. Verifying these names can help reduce potential errors in your PST mapping file.
 
-The Microsoft Azure Storage Explorer is in Preview.
+The Microsoft Azure Storage Explorer desktop standalone application is generally available. You can download the latest version using the link provided below. There is an option of using Azure Storage Explorer via Azure Portal. However, it is currently in preview.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-network-upload-to-import-pst-files.md]:
```suggestion
The Microsoft Azure Storage Explorer desktop standalone application is generally available. You can download the latest version using the link provided below. There is also an option of using the Azure Storage Explorer via the Azure Portal. However, it is currently in preview.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@ShraddhaSSwadi Please review the suggested changes and update the PR. Thanks

**Comment by markjjo** [comment]
@ShraddhaSSwadi I would rather that we not mention accessing the Azure Storage Explorer via the Azure portal. This is out of scope for PST Import in my opinion (because customer uploads to MS-provided storage blob). Also, wouldn't customer access from their of Azure Storage account? I plan to remove this sentence (and that it is only in preview). For PST Import scenario, the goal of this step is to verify that PSTs were uploaded. The standalone version accomplishes this. Please ping my offline if you want to discuss.

**Pull request merged**